index.html

```diff
@@ -326,9 +338,63 @@ <h3>Adhere to the rules of ARIA</h3>
+        <p>
+          Additionally, ARIA specifically mentions in <a data-cite="wai-aria-1.2/#host_general_conflict">Conflicts with Host Language Semantics</a> 
+          that if authors use both native HTML features for exposing states and properties as well as their ARIA counterparts, then
+          the host language features take priority over the explicit ARIA attributes that are also used.
+        </p>
+        <p>
+          For instance, in the following example an author is using HTML's `input type=checkbox` and has specified an `aria-checked=true`. However,
+          user agents are meant to ignore the `aria-checked` attribute. Instead user agents would expose the state based on the native features
+          of the form control.
+        </p>
+        <pre class="HTML example" title="The implicit checked state takes precedent over the explicit ARIA attribute">
+          &lt;!-- Do not do this! -->
+          &lt;input type="checkbox" checked aria-checked="false">
+        </pre>
+      </section>
+      <section>
+        <h3>Adhere to the rules of HTML</h3>
+        <p>
+          While ARIA can be used to alter the way HTML features are exposed to users of assistive technologies,
+          these modifications do not change the underlying parsing and allowed content models of HTML. For instance,
+          a [^div^] allows an author to specify any role on it. However, this does not mean that the element can then be
+          used in a way that deviates from the rules HTML has defined for the element.
+        </p>
+        <p>
+          For instance, in the following example an author has specified a role of `link` on a `div` element. While
+          HTML allows for a `link` element to be a child of a `p` element, the HTML does not allow a `div` within
+          a `p` element. 
+        </p>
+        <pre class="HTML example" title="Revised ARIA semantics with invalid HTML nesting">
+          &lt;!-- Do not do this! -->
+          &lt;p>
+            ... &lt;div role=link tabindex=0>...&lt;/div> ... 
+          &lt;/p>;
+        </pre>
+        <p>
+          The HTML parser will render the above markup as the following:
+        </p>
+        <pre class="HTML example" title="Unwanted rendered markup with valid alternative solution">
+          &lt!-- The previous example's markup will render as follows -->
+          &lt;p>...&lt;/p>
+          &lt;div role=link tabindex=0>...&lt;/div> 
+          ... 
+          &lt;p>&lt;/p>;
+          
+          &lt;!-- Use a span are allowed in p elements! -->
+          &lt;p>
+            ... &lt;span role=link tabindex=0>...&lt;/span> ...
+          &lt;/p>
+        </pre>
+        <p>
+          While this specification indicates the allowed ARIA attributes which can be specified on each HTML element,
+          this example illustrates that even if a role is allowed, the context in which it is used can still result
+          in a rendering and potentially accessibility issues.
```

Comment:
"a rendering and potentially accessibility issues."

suggest either
"rendering and potentially accessibility issues."
or
"a rendering issue and, potentially, accessibility issues."